src/juicy-popover.html

```diff
@@ -209,93 +108,208 @@
 
             return newExpandableRect;      
         }
+            
+        class JuicyPopoverElement extends HTMLElement {
+            
+            get disabled(){
+                return this._disabled;
+            }
+            
+            set disabled(val) {
+                if (this._disabled !== val) {
+                    this._disabled = val;
+                    if (val) {
+                        this._expanded = false;
+                        this.setAttribute('disabled', '');
+                    }
+                    else {
+                        this.removeAttribute('disabled');
+                    }
+                }
+            }
+            
+            get expanded(){
+                return this._expanded;
+            }
+            
+            set expanded(val) {
+                if (this._expanded !== val) {
+                    this._expanded = val;
+                    if(val) {
+                        this.setAttribute('expanded', '');
+                        this._expand();
+                    } else {
+                        this.removeAttribute('expanded');
+                        this._collapse();
+                    }
+                }
+            }
```

Comment:
I'd rather `set-`/`removeAttribute` in setters (`getAttribute` in getter), then you may not need those `_properties` which are unnecessarily exposing yet another internal to the API. Maybe you could also simplify `if`s a little bit (I suppose they are only to avoid infinite loop in `attributeChangedCallback`).